routes/cat.go

```diff
@@ -15,20 +15,39 @@ func getCatAll(c web.C, w http.ResponseWriter, r *http.Request) {
 	var err error
 	sts, err := st.FindAll(ds.Where{"id", ">", 0}, 0)
 	panicOnErr(err)
-	jsn, err := json.Marshal(sts)
-	fmt.Fprintf(w, string(jsn))
+
+	catsJSON, err := json.Marshal(sts)
+	if err != nil {
```

Comment:
Don't forget about DRY principle
```go
if err != nil {
		res := "{'result': 'failure'}"
		result, err := json.Marshal(res)
		fmt.Fprintf(w, string(result))
		panicOnErr(err)
	} else {
		fmt.Fprintf(w, string(catsJSON))
	}
```
same code you wrote many times, how about to create function to encapsulate this functionality